integrations/cache/cache-compat.php

```diff
@@ -70,7 +72,16 @@ public function add_cookie_script() {
 
 		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : ' type="text/javascript"';
 
-		echo "<script{$type_attr}>\n{$js}\n</script>\n"; // phpcs:ignore WordPress.Security.EscapeOutput
+		$script = "<script{$type_attr}>\n{$js}\n</script>\n";
+
+		/**
+		 * Filters the content of the script tag that creates the cookie.
+		 *
+		 * @since 3.7
+		 *
+		 * @param string $script The Script tag and content.
+		 */
+		echo apply_filters( 'pll_cookie_script', $script ); // phpcs:ignore WordPress.Security.EscapeOutput
```

Comment:
```suggestion
		echo apply_filters( 'pll_cookie_script', "<script{$type_attr}>\n{$js}\n</script>\n" ); // phpcs:ignore WordPress.Security.EscapeOutput
```
If you chose to apply my suggestion, please delete previous `$script` assignation.